### PR TITLE
assignment for blocktest hands on - added new class ActionPrint that…

### PR DESCRIPTION
**Pull request by davidetome**
added ActionPrint class to print a string in the shell

### CODE REVIEW

File: src/genericactiondepotlib/src/actionPrint.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ */
+class ActionPrint : public Action
+{
+    public:
+        ActionPrint(const CommandAttributes& commandAttributes,const std::string& testCode);    
+        execution execute(const TestRepetitions& testrepetition) override;
+        void beforeExecute() override;
+
+    private:        
+        std::string value_;
```
Comment (triccyx) [inline src/genericactiondepotlib/src/actionPrint.h]: Use a more significative name for the variable ie. message_

Comment (triccyx) [inline src/genericactiondepotlib/src/actionPrint.h]: Use a more significative name for the variable ie. message_

**Review by triccyx — CHANGES_REQUESTED** [review]

**Comment by triccyx** [comment]
Please also add an xml file in src/scriptbuilder/xmltemplate/ general

File: src/genericactiondepotlib/src/actionPrint.h (hunk 1 above)
Comment (davidetome) [reply src/genericactiondepotlib/src/actionPrint.h]: done

Comment (triccyx) [reply src/genericactiondepotlib/src/actionPrint.h]: Also this Please also add an xml file in src/scriptbuilder/xmltemplate/general

Comment (davidetome) [reply src/genericactiondepotlib/src/actionPrint.h]: done

**Review by triccyx — APPROVED** [review]

**Review by triccyx — APPROVED** [review]

**Pull request merged**